components/badge-first-run/badge_first_run.c

```diff
@@ -29,7 +29,7 @@
 #include <badge_disobey_samd.h>
 #include <badge_erc12864.h>
 #include <font.h>
-#include <sha2017_ota.h>
+#include <badge_ota.h>
```

Comment:
Is dit niet gewoon stiekem een generieke OTA functie? Waarom noemen wij het niet gewon ota?

Reply:
Ja . . ik had wat "ruzie" en dacht dat het door de naamgeving kwam . .
Aan andere kant .. is het wel "badge.team specifiekige" OTA . . toch?